draft-ietf-lake-edhoc.md

```diff
@@ -789,7 +789,7 @@ PRK_exporter  = EDHOC-KDF( PRK_out, 10, h'', hash_length )
 
 where hash_length denotes the output size in bytes of the EDHOC hash algorithm of the selected cipher suite.
 
-PRK_exporter MUST be derived anew from PRK_out if EDHOC-KeyUpdate is used, see {{keyupdate}}.
+PRK_exporter MUST be derived anew from PRK_out if EDHOC-KeyUpdate is used, see {{keyupdate}}. A typical mode of operation of an application is that PRK_exporter is derived from PRK_out generated by EDHOC, and application keys in turn from PRK_exporter. Later, when key update is required by the application, PRK_out is updated  from which new PRK_exporter and application keys are derived.
```

Comment:
"A typical mode of operation of an application is that PRK_exporter is derived from PRK_out generated by EDHOC, and application keys in turn from PRK_exporter."

Isn't this always the case?